db-engine/src/main/java/com/splicemachine/db/impl/sql/execute/DecimalBufferedSumAggregator.java

```diff
@@ -81,7 +83,7 @@ public void merge(ExecAggregator addend) throws StandardException {
 				if(addend==null) return; //treat null entries as zero
 				//In Splice, we should never see a different type of an ExecAggregator
 				DecimalBufferedSumAggregator other = (DecimalBufferedSumAggregator)addend;
-            if (other.isNull){
+            if (other.isNull && other.position == 0){
```

Comment:
Why do we need to add the check of other.position == 0? So other.isNull is not enough to represent that other is  empty?

Reply:
I was trying to handle the case where we overflow during incrementPosition(), and so isNull got set to false, but the new value never got added, so the total sum should still be null.  But now I see this check is bad because position could be 0 when sum is non-zero.  I think it can be fixed by delaying the setting of isNull in incrementPosition().  I will make the change.